### PR TITLE
Added troubleshooting steps to view generated code

### PR DESCRIPTION
**Pull request by Massimo37**
Added troubleshooting steps to view generated code from a CI build

### CODE REVIEW

File: doc/articles/uno-development/debugging-uno-ui.md [hunk 1]
```diff
@@ -103,6 +103,19 @@ The Source Generation tooling diagnostics can be enabled as follows:
...
 **Make sure to remove the `UnoSourceGeneratorUnsecureBinLogEnabled` property once done.**
 
+If ever the need arises to view the generated source code of a *failing* CI build, you can perform the following steps:
+
+1. In your local branch, locate the one of build yaml files (located in the root Uno folder):
+     - .azure-devops-android-tests.yml
+     - .azure-devops-macos.yml
+     - .azure-devops-wasm-uitests.yml
+    
+2. At the bottom of the yaml files, you'll find *Publish...* tasks, right above these tasks, copy/paste the following code to create a task which will copy all generated source files and put them in an artifact for you to download
```
Comment (benventive) [inline doc/articles/uno-development/debugging-uno-ui.md]: I believe the "following code" is missing

Comment (Massimo37) [reply doc/articles/uno-development/debugging-uno-ui.md]: done

File: doc/articles/uno-development/debugging-uno-ui.md [hunk 2]
```diff
@@ -103,6 +103,23 @@ The Source Generation tooling diagnostics can be enabled as follows:
...
+If ever the need arises to view the generated source code of a *failing* CI build, you can perform the following steps:
+
+1. In your local branch, locate the one of build yaml files (located in the root Uno folder):
+     - .azure-devops-android-tests.yml
+     - .azure-devops-macos.yml
+     - .azure-devops-wasm-uitests.yml
+    
+2. At the bottom of the yaml files, you'll find *Publish...* tasks, right above these tasks, copy/paste the following code to create a task which will copy all generated source files and put them in an artifact for you to download:
+
+       - bash: cp -r $(build.sourcesdirectory)/src/SamplesApp/SamplesApp.Droid/obj/Release/g/XamlCodeGenerator/ $(build.artifactstagingdirectory)
```
Comment (benventive) [inline doc/articles/uno-development/debugging-uno-ui.md]: We should probably indicate to change this path depending on the platform that's failing (Droid/iOS)

Comment (Massimo37) [reply doc/articles/uno-development/debugging-uno-ui.md]: done

**Pull request merged**